Client.xcodeproj/project.pbxproj

```diff
@@ -7657,8 +7660,7 @@
 				4422D4E121BFFB7600BF1855 /* filter_block.cc in Sources */,
 				CA2EE0A727349F760089C75F /* OnboardingAdblockDisconnect.swift in Sources */,
 				0A8C69BE225E350300988715 /* IndentedImageTableViewCell.swift in Sources */,
-				5E096180252B63F200F3AFBB /* BraveSyncAPI+Utilities.swift in Sources */,
-				0AE5C69124F0059D004CBC9B /* OnboardingPrivacyConsentViewController.swift in Sources */,
```

Comment:
this removal is weird, can you double check it's correct?

Reply:
These are the stale files that we forgot to remove from settings.